trim_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './base_decorator'
+
+class TrimDecorator < Decorator
+  def correct_name
+    @nameable.correct_name[0..9]
```

Comment:
- [ OPTIONAL ] Great job working on the required changes but I think it would be great if you add the condition that the output of @nameable.correct_name has a maximum of 10 characters. If it's longer it should trim the word.  so if the word longer than 10 chars so trim the word so please add the condition 🤩⭐🚀 and if it is not just return the word ⭐and  I am assiging it as optional because the previous reviewer didn't mention it but it still important to fix it 💪🏻